pytools/__init__.py

```diff
@@ -1489,10 +1489,12 @@ class Table:
     .. automethod:: add_row
     .. automethod:: __str__
     .. automethod:: latex
+    .. automethod:: github_markdown
     """
 
-    def __init__(self):
+    def __init__(self, alignments=['l', 'r']):
         self.rows = []
+        self.alignments = alignments
```

Comment:
Not sure. I was reluctant to change the format of the default output via `__str__`, since someone might already rely on the existing format. For `latex()` it won't make a difference, since there is no alignment.

Reply:
Could you pick the default alignments in such a way that they don't change the behavior?

Reply:
Changed default alignments to `['l']`, which should match the current behavior.